teerex/src/lib.rs

```diff
@@ -92,6 +92,12 @@ pub mod pallet {
 			hash: H256,
 			data: Vec<u8>,
 		},
+		TcbInfoRegistered {
+			fmspc: Fmspc,
+		},
+		QuotingEnclaveRegistered {
+			enclave_identity: Vec<u8>,
+		},
```

Comment:
Conclusively I think this should be as follows:
```rust
TcbInfoRegistered {
	fmspc: Fmspc,
        tcb_info: TcbInfoOnChain
},
QuotingEnclaveRegistered {
	enclave_identity: QuotingEnclave,
},
```

Is there any downside of putting the whole data @brenzi? I don't think so.